include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -9,7 +9,7 @@
  *
  * ONNX Runtime's C, C++ APIs offer an easy to use interface to onboard and execute onnx models.
```

Comment:
* ONNX Runtime is a high-performance inference and training graph execution engine for deeplearning models.
--> 

 * ONNX Runtime is a high-performance inference and training graph execution engine for deep learning models.